### ISSUE TITLE
Can't stop two nodes concurrently with wait_other_notice=True.

### ISSUE THREAD

**Issue by nyh** [scylladb/scylla-ccm#124]
The dtest "materialized_views_test.TestMaterializedViews.double_node_failure_during_mv_insert_4_nodes_test" starts 4 nodes, and kills two of them with wait_other_notice=True.
The log contains something like this:
```
2019-01-17 14:42:10.215307 START: stop node node3
2019-01-17 14:42:10.215669 START: stop node node2
2019-01-17 14:42:17.325150 FINISH: stop node node2
```
The "stop node3" never finished, 
and eventually produces
```
TimeoutError: 17 Jan 2019 12:52:18 [node2] Missing: ['127.0.0.3 is now [dead|DOWN]']:
```
What happens is that we stop node3, and then "wait for node2 to notice", which means grepping node2's log for message about node3 becoming dead. But since node2 has also died, it will obviously never notice.

The "waiting for notice" should stop waiting for a certain node's log if that node itself dies.

**Comment by nyh** [comment]
The new `other_nodes` parameter added in https://github.com/scylladb/scylla-ccm/commit/7238928ea5c3509a05d931d4091d4b44445f3934 allows to solve this issue by listing only the nodes to wait for, and this list should not contain either node which we are planning to take down in the test.

So we should perhaps close this issue - even though the committed workaround isn't as automatic and fool-proof as it could have been and this issue described?

**Comment by bhalevy** [comment]
Can be closed with eb2f95a97ca679e85c313e3b3a803c65a2d365b1
